DeepSpeech.ipynb

```diff
@@ -86,17 +86,34 @@
     "from math import ceil\n",
     "import tensorflow as tf\n",
     "from util.log import merge_logs\n",
+    "from util.generic import check_so\n",
     "from util.gpu import get_available_gpus\n",
     "from util.text import sparse_tensor_value_to_texts, wers\n",
     "from tensorflow.python.ops import ctc_ops\n",
+    "from tensorflow.python.client import timeline\n",
     "\n",
     "ds_importer = os.environ.get('ds_importer', 'ted_lium')\n",
     "ds_dataset_path = os.environ.get('ds_dataset_path', './data/ted')\n",
     "\n",
     "import importlib\n",
     "ds_importer_module = importlib.import_module('util.importers.%s' % ds_importer)\n",
     "\n",
-    "from util.website import maybe_publish"
+    "from util.website import maybe_publish\n",
+    "\n",
+    "# Switch env var ds_do_fulltrace to something not an empty string\n",
```

Comment:
Loading `libcupti.so` makes the entire code platform dependent, it will now run without a warning linux if I want to do a full trace. However, on an os x machine with `libcupti.dylib` present is will falsely complain 
```
WARNING: Running on GPU but no libcupti.so could be found ; will be unable to report GPU VRAM usage
```
I'd rather avoid this and take this code that checks for `libcupti.so` out.